MarkSkalskyi/README.md

```diff
@@ -0,0 +1,42 @@
+# Getting started with [HDRezka](https://rezka.ag)
+
+HDRezka is a website that allows you to watch your favorite movies and series. If you need a guide for signing in, you're in the right place.
+
+**Movies? HDRezka!**
+
+## Step 1: Registration
+
+1. Firstly, visit the HDRezka website at [HDRezka](https://rezka.ag).
+   This is how it looks like:
+   ![Screenshot of the HDRezka website](https://ibb.co/JxMxSZ1)
```

Comment:
Your screenshot links don't work. No images are displayed in the rendered README.
![image](https://github.com/ivancheban/kpi-tech-writer-course/assets/47241364/a381a6ad-cb6d-4c71-9a4b-fbb8cfda8998)

Reply:
To add images to your README.md file, either add the images folder next to your README.md file, or provide the links to a valid screenshot image on the internet. The first option is more preferable. For example, if you add the `images` folder next to your README file and add the `images/signup.png` screenshot, you can reference this screenshot in the README.md file: \[Signup](images/signup.png)